src/store/ChartStore.js

```diff
@@ -307,6 +311,15 @@ class ChartStore {
         this.stxx = null;
     }
 
+    @action.bound onReceiveProps(nextProps) {
+        if (!this.context) return;
+
+        const { symbol } = nextProps;
+        if (symbol && this.activeSymbols.length > 0 && this.currentActiveSymbol.symbol !== symbol) {
+            this.changeSymbol(symbol);
```

Comment:
I can't find `onSymbolChange()` getting fired in `this.changeSymbol(symbol)`.

Are we sending the notification?

Reply:
@aminroosta no - it is called by the `symbolChange` event from the chart engine: https://github.com/binary-com/SmartCharts/blob/dev/src/store/ChartStore.js#L251

`changeSymbol` fires this event when it creates a new chart.